R/getRunStatus.R

```diff
@@ -126,8 +126,15 @@ getRunStatus <- function(mydir = dir(), sort = "nf", user = NULL) {
         miffile <- paste0(ii, "/validation.mif")
         out[i, "Mif"] <- if (file.exists(miffile) && file.info(miffile)[["size"]] > 99999) "yes" else "no"
       } else {
-        miffile <- paste0(ii, "/REMIND_generic_", cfg[["title"]], ".mif")
-        out[i, "Mif"] <- if (file.exists(miffile) && file.info(miffile)[["size"]] > 3899999) "yes" else "no"
+        miffile    <- paste0(ii, "/REMIND_generic_", cfg[["title"]], ".mif")
+        sumErrFile <- paste0(ii, "/REMIND_generic_", cfg[["title"]], "_summation_errors.csv")
+        if (file.exists(sumErrFile)){ 
```

Comment:
Maybe a stupid question, but might it be possible that the `sumErrFile` exists, but the reporting then somehow failed?
So maybe would it be better to do
```
if (! file.exists(miffile)) {
  out[i, "mif"] <- "no"
} else if (file.exists(sumErrFile)) {
  out[i, "mif"] <- "sumErr"
} else {
  out[i, "mif"] <- "yes"
}
```

Reply:
Good point, I changed the conditions as proposed.